platform/viewer/.webpack/webpack.pwa.js

```diff
@@ -35,7 +35,7 @@ const setHeaders = (res, path) => {
   }
   if (path.indexOf('.pdf') !== -1) {
     res.setHeader('Content-Type', 'application/pdf');
-  } else if (path.indexOf('/frames') !== -1) {
+  } else if (path.indexOf('frames') !== -1) {
```

Comment:
Fails on windows, causing e2e server test failures, so I needed this to test locally.